viadot/sources/genesys.py

```diff
@@ -160,6 +160,9 @@ def genesys_generate_exports(
         Args:
             post_data_list (List[str], optional): List of string templates to generate json body. Defaults to None.
             end_point (str, optional): Final end point for Genesys connection. Defaults to "reporting/exports".
+
+        Returns:
+            Union[None, dict]: Dict when the "conversations" endpoint is called, otherwise returns None.
```

Comment:
I cannot find the second `return`. You wrote that it should be `Dict` or `None`

Reply:
changed by Optional, as it is said in python documentation, it should be the best practice.